script.js

```diff
@@ -0,0 +1,116 @@
+const inputJs = document.querySelector(".input"),
+    toDoNewJs = inputJs.querySelector("#todo-new"),
+    listWaitingJs = document.querySelector(".waiting"),
+    listCompleteJs = document.querySelector(".complete");
+
+const TODOS = "toDos"; //key for saving todos in local storage
+
+let toDos = [];
+let maxId = 0;
+let countWaiting = 0, countComplete = 0;
+let tmpType; //only for deleteToDo and moveToDo
+
+function saveToDos() {
+    localStorage.setItem(TODOS, JSON.stringify(toDos));
+}
+
+function printCount() {
+    const countWaitingJs = document.querySelector("#count-waiting"),
+        countCompleteJs = document.querySelector("#count-complete");
+    countWaitingJs.innerText = countWaiting;
+    countCompleteJs.innerText = countComplete;
+}
+
+function deleteToDo(event){
+    const li = event.target.parentNode;
+    tmpType = li.parentNode.className;
+    
+    if(tmpType === "list waiting"){ 
+        listWaitingJs.removeChild(li);
+        countWaiting -= 1; 
+    } 
+    else{
+        listCompleteJs.removeChild(li);
+        countComplete -= 1;    
+    }
+    printCount();
+   
+    const cleanToDos = toDos.filter(function(toDo){
+        return toDo.id !== parseInt(li.id);
+    });
+    toDos = cleanToDos;
+    saveToDos();
+
+    maxId += 1; //prevent ovelapping of id
+}
+
+function moveToDo(event){
+    deleteToDo(event);
+    const targetContent = event.target.innerText;
+    if(tmpType === "list waiting") { printToDo(targetContent, "comp"); }
+    else { printToDo(targetContent, "wait");}
+}
+
+function printToDo(content, type) {
```

Comment:
```suggestion
function addToDo(content, type) {
```
content, type 두 가지 파라미터 이용해서 printToDo 짜신 것 너무 좋네요.
printToDo 함수명이 출력한다는 의미랑 헷갈리게 느껴져서 addToDo 같은 것도 좋은 것 같아요 !

Reply:
완전 맞는 말이세요!!11 addTodo로 고쳤어요 ㅎㅎ